src/handler.py

```diff
@@ -144,8 +144,18 @@ def control_db_utilization(event, context):
     if stage not in STAGES:
         raise Exception(f"stage not recognized {os.getenv('STAGE')}")
     if alarm_state == "ALARM":
-        logger.info(f"Disabling trigger {TRIGGER[stage]} because error handler is in alarm")
-        disable_trigger(event, context)
+        logger.info(f"ALARM!")
```

Comment:
:point_up: -- I'd vote for renaming this function.

Reply:
Okay, I'll do that.  We all call it circuit breaker anyway.